### PR TITLE
[strings] use "station building" instead of "train station"

### PR DESCRIPTION
**Pull request by RedAuburn**
before/after
image1

### CODE REVIEW

**Comment by RedAuburn** [comment]
@organicmaps/translations please check

**Review by map-per — APPROVED** [review]
Looks good to me and the German string is good.

**Review by matheusgomesms — APPROVED** [review]

**Review by vng — APPROVED** [review]

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -7115,43 +7115,47 @@
...
+    it = Edificio della stazione
+    ja = 駅ビル
+    ko = 역 빌딩
+    lt = Stoties pastatas
+    mr = स्टेशन बिल्डिंग
+    nb = Stasjonsbygningen
+    nl = Stationsgebouw
+    pl = Budynek stacji
+    pt = Edifício da estação
+    ro = Clădirea gării
```
Comment (vgrigoriu) [inline data/strings/types_strings.txt:7148]: Not sure why this change was made, "Station Building" doesn't sound natural to me, but then I'm not a native English speaker. Anyway, for Romanian the old translation was clunky, but the new one is not much better, I suggest simply:
```suggestion
    ro = Gară
```

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -7115,43 +7115,47 @@
...
+    da = Stationsbygning
+    de = Bahnhofsgebäude
+    el = Κτίριο σταθμού
+    es = Edificio de la estación
+    et = Jaamahoone
+    eu = Geltoki Eraikina
+    fa = ساختمان ایستگاه
+    fi = Asemarakennus
+    fr = Bâtiment de la gare
+    he = בניין התחנה
```
Comment (NeatNit) [inline data/strings/types_strings.txt:7135]:
```suggestion
    he = בניין תחנה
```

(I also don't see why make this change)

**Comment by jimcarst** [comment]
Nl translation is OK, but no one says _station building_ over here, only _station_.

**Comment by map-per** [comment]
I think the confusion here is that the new translation is not for the Train Station POI (railway=station), but for the building usage (building=train_station). The POI has a [separate translation](https://github.com/organicmaps/organicmaps/blob/381cd93580f51cde799062abfee09f1dd6ceeaff/data/strings/types_strings.txt#L23154-L23191).

So the new category string is not for train stations, but for buildings that are used as a train station.

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -7115,43 +7115,47 @@
...
+    ja = 駅ビル
+    ko = 역 빌딩
+    lt = Stoties pastatas
+    mr = स्टेशन बिल्डिंग
+    nb = Stasjonsbygningen
+    nl = Stationsgebouw
+    pl = Budynek stacji
+    pt = Edifício da estação
+    ro = Clădirea gării
+    ru = Здание вокзала
```
Comment (pastk) [inline data/strings/types_strings.txt:7149]: Здание ж/д станции

**Review by pastk — APPROVED** [review]
Related:

Why do we have a special type for `building=train_station`, while all other `building=*` are just buildings?

We render them the same, the only difference is translation.
Shall we add specific translations for other `building=*` options?

File: data/strings/types_strings.txt (hunk 1 above)
Comment (vgrigoriu) [reply data/strings/types_strings.txt:7148]: > the new translation is not for the Train Station POI (railway=station), but for the building usage (building=train_station)

That seems like an extremely subtle distinction, I wouldn't know when to use `building=train_station`. But anyway, in this case `Clădirea gării` is fine for Romanian.

**Comment by pastk** [comment]
> So the new category string is not for train stations, but for buildings that are used as a train station.

Actually its for buildings **built** as a train station. Sometimes there is no current/active train station (but e.g. a restraunt instead), hence the distinction between a current amenity `railway=station` and a physical structure `building=train_station`.
Same goes to e.g. `building=church` vs `amenity=place_of_worship`, etc.

**Comment by RedAuburn** [comment]
> I am not sure if I am following the idea. What was wrong with the "Train Station"?

This string is for ways tagged with only `building=train_station`, `railway=station` still uses the "train station" string. 

In search, multiple features with the type name "Train Station" are returned, but most are actually `building=train_station`. This change differentiates the buildings from the actual train station.

**Comment by vng** [comment]
Please rebase and I will merge it.

File: data/strings/types_strings.txt (hunk 1 above)
Comment (RedAuburn) [reply data/strings/types_strings.txt:7167]: updated the railway.station translation to Gară, thanks 👍️

File: data/strings/types_strings.txt (hunk 2 above)
Comment (RedAuburn) [reply data/strings/types_strings.txt]: thanks 👍️

File: data/strings/types_strings.txt (hunk 3 above)
Comment (RedAuburn) [reply data/strings/types_strings.txt]: thanks 👍️

**Comment by RedAuburn** [comment]
sorry for the long delay, have updated translations based on the comments & rebased :) @vng

**Comment by RedAuburn** [comment]
improved basque translation (https://github.com/organicmaps/organicmaps/commit/4657043b56bbe67c81de8bf21bb9f184df8098ae#commitcomment-148672536)
